project/Release.scala

```diff
@@ -18,19 +16,7 @@ object Release {
         "git@github.com:allenai/sbt-plugins.git"
       )
     ),
-    // Publish settings.
-    // http://www.scala-sbt.org/0.12.2/docs/Detailed-Topics/Artifacts.html
-    publishArtifact := true, // Enable publish
     publishArtifact in Test := false,
-    publishMavenStyle := false,
```

Comment:
Do you know if the default is false off the top of your head?
Looking at https://www.scala-sbt.org/1.x/docs/Publishing.html#Modifying+the+generated+POM I would _guess_ it is false but not sure at all.

Reply:
It's set by the codeartifact plugin.

Reply:
So it looks like we're changing the setting right?
https://github.com/bbstilson/sbt-codeartifact/blob/397b52921f6fbfd22e6fc2923c14232dd1878069/sbt-codeartifact/src/main/scala/codeartifact/CodeArtifactPlugin.scala#L49

Is this a CodeArtifact requirement or something else? Not objecting to it, just trying to understand it.